rwengine/src/render/ObjectRenderer.cpp

```diff
@@ -275,11 +275,12 @@ void ObjectRenderer::renderVehicle(VehicleObject* vehicle,
         auto& wi = vehicle->physVehicle->getWheelInfo(w);
         // Construct our own matrix so we can use the local transform
         vehicle->physVehicle->updateWheelTransform(w, false);
+        bool isRhino = (vehicle->getVehicle()->vehiclename_ == "RHINO");
```

Comment:
nit: A comment wouldn't hurt why this applies to rhino

aside from that also LGTM without having tested it.